crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -346,6 +363,8 @@ impl NetworkConfig {
 }
 
 impl Config {
+    const DEFAULT_HEAD_POLL_INTERVAL: std::time::Duration = std::time::Duration::from_secs(30);
```

Comment:
Ah I see its because of mainnet vs other networks. I think lets make all networks just 30s now. The 300s is a remnant from when mainnet blocks were published eevery 2 hours! The only reason the current limit hasn't bothered us is because those that matter run with pending enabled.

Reply:
Made a default one of 30s.